src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,542 @@
+            ->setReplaceToken('{body}', (string)$bodyReplace)
+            ->setReplaceToken('{appName}', $projectDirectory)
+            ->buildFile($destinationFile);
+
+        return 0;
+    }
+
+    /**
+     * Build a dependencies file from the template.
+     *
+     * @param string     $templatePath        Template file path.
+     * @param string     $destinationFile     Destination file to write to.
+     * @param Dependency $dependencyContainer Dependency Container Dependency.
+     *
+     * @return int The Exit Code.
+     */
+    private function buildDependenciesFile(
+        string $templatePath,
+        string $destinationFile,
+        Dependency $dependencyContainer
+    ): int {
+        $importsReplace = null;
+        $argumentReplace = null;
+        $bodyReplace = null;
+
+        switch (get_class($dependencyContainer)) {
+            case PHPDIDependency::class:
+                $importsReplace = "\nuse DI\ContainerBuilder;\nuse Monolog\Handler\StreamHandler;\n" .
+                    "use Monolog\Logger;\nuse Monolog\Processor\UidProcessor;\n" .
+                    "use Psr\Container\ContainerInterface;\nuse Psr\Log\LoggerInterface;\n";
+                $argumentReplace = 'ContainerBuilder $containerBuilder';
+                $bodyReplace = file_get_contents(
+                    $this->templatesDirectory . DIRECTORY_SEPARATOR . 'parts' .
+                    DIRECTORY_SEPARATOR . 'dependencies_body_php_di.template'
+                );
+                break;
+            case PimpleDependency::class:
+                $importsReplace = "\nuse Monolog\Handler\StreamHandler;\nuse Monolog\Logger;" .
+                    "\nuse Monolog\Processor\UidProcessor;\nuse Pimple\Container;\nuse Psr\Log\LoggerInterface;\n";
+                $argumentReplace = 'Container $container';
+                $bodyReplace = file_get_contents(
+                    $this->templatesDirectory . DIRECTORY_SEPARATOR . 'parts' .
+                    DIRECTORY_SEPARATOR . 'dependencies_body_pimple.template'
+                );
+                break;
+            case OtherDependency::class:
+                $importsReplace = '';
+                $argumentReplace = '$container';
+                $bodyReplace = "\n";
+                break;
+        }
+
+        (new FileBuilder($templatePath))
+            ->setReplaceToken('{imports}', $importsReplace)
+            ->setReplaceToken('{argument}', $argumentReplace)
+            ->setReplaceToken('{body}', (string)$bodyReplace)
+            ->buildFile($destinationFile);
+
+        return 0;
+    }
+
+    /**
+     * Build a index file from the template.
+     *
+     * @param string     $templatePath        Template file path.
+     * @param string     $destinationFile     Destination file to write to.
+     * @param Dependency $dependencyContainer Dependency Container Dependency.
+     *
+     * @return int The Exit Code.
+     */
+    private function buildIndexFile(
```

Comment:
Would each profile have to reinvent this method in order to be able to maybe just add one line of code (for example to add a twig middleware)?

Reply:
I can make them `protected`.